all-in-one/Dockerfile

```diff
@@ -87,6 +81,6 @@ ENV PERSISTENCE_FILE_PARENT_PATH="oscal-content" \
 
 EXPOSE 8080
 
-WORKDIR /app/
+WORKDIR /app
 
-ENTRYPOINT ["java", "-jar", "oscal-rest-service-app.jar"]
\ No newline at end of file
+CMD ["oscal-rest-service-app.jar"]
```

Comment:
Okay so this is viable because the upstream image already sets `ENTRYPOINT` during the build? Just want to make sure I have some understanding of this.

Reply:
Yes, the java distroless images set the entrypoint to "java -jar" such that all you need to do when using the image is to supply the location of the jar file. See the following references:
- [java distroless image build definition](https://github.com/GoogleContainerTools/distroless/blob/4b83f1595ffbf6f3b658f15d06bc0936fc8fa57a/java/BUILD#L88)
- [java distroless image usage instructions](https://github.com/GoogleContainerTools/distroless/tree/main/java#usage)